src/locales/sl/translation.json

```diff
@@ -0,0 +1,238 @@
+      "open": "Odpri denarnico"
+    },
+    "privateKey": {
+      "header": "Vnesite zasebni ključ",
+      "instruction": "Vnesite vaš zasebni ključ, zakodiran v načinu base-64.",
+      "error": "Neveljaven zasebni ključ",
+      "enterPrivateKeyHere": "Zasebni ključ vnesite tu"
+    },
+    "ledger": {
+      "selectWallets": "Izberite denarnice",
+      "cancel": "Prekliči",
+      "openWallets": "Odpri"
+    },
+    "bitpie": {
+      "warning": "❗ Uporabniki denarnice BitPie: Uvoz vašega mnemonika iz BitPie neposredno v Oasis Wallet ni podprto. <0>Več informacij o težavi dobite v dokumentaciji.</0>"
+    }
+  },
+  "createWallet": {
+    "newMnemonic": "Ustvari nov mnemonik",
+    "thisIsYourPhrase": "To je vaš mnemonik",
+    "instruction": "Ključne besede mnemonika <1>v pravilnem vrstnem redu</1> shranite na varno mesto, saj jih boste potrebovali vsakič, ko boste želeli odpreti denarnico.",
+    "doNotShare": "Nikoli ne delite ključnih besed, saj lahko z njimi kdor koli dostopa do vaše denarnice in kovancev.",
+    "confirmSaved": "Ključne besede sem shranil",
+    "mnemonicMismatch": "Entered mnemonic does not match."
+  },
+  "account": {
+    "summary": {
+      "yourAccount": "To je vaš račun.",
+      "notYourAccount": "To ni vaš račun.",
+      "balance": {
+        "total": "Stanje",
+        "available": "Na voljo",
+        "debonding": "V razvezi",
+        "delegations": "Vloženo"
+      },
+      "noTransactionFound": "Ni transakcij"
+    },
+    "transaction": {
+      "transfer": {
+        "received": "Prejeto <0></0>",
+        "sent": {
+          "header": "Poslano <0></0>"
+        }
+      },
+      "addEscrow": {
+        "received": "Prejeto <0></0> sredstev v delegiranje",
+        "sent": "Delegiranih <0></0> overiteljem"
+      },
+      "reclaimEscrow": {
+        "received": "<0></0> prejetih od delegatorja",
+        "sent": "Vrnjenih <0></0> s strani overitelja"
+      },
+      "loadingError": "Transakcij ni bilo mogoče pridobiti. Seznam je zato verjetno prazen ali zastarel."
+    },
+    "sendTransaction": {
+      "recipient": "Prejemnik",
+      "enterAddress": "Vnesite naslov",
+      "success": "Transakcija uspešno izvršena. Novo stanje na vašem računu bo prikazano v nekaj trenutkih.",
+      "send": "Pošlji"
+    },
+    "loading": "Nalaganje računa",
+    "addEscrow": {
+      "delegate": "Delegiraj"
+    },
+    "subnavigation": {
+      "transactions": "Transakcije",
+      "activeDelegations_0": "Aktivne delegacije ({{count}})",
+      "activeDelegations_1": "Aktivne delegacije ({{count}})",
+      "activeDelegations_2": "Aktivne delegacije ({{count}})",
+      "activeDelegations_3": "Aktivne delegacije ({{count}})",
+      "debondingDelegations_0": "Delegacije v razvezi ({{count}})",
+      "debondingDelegations_1": "Delegacije v razvezi ({{count}})",
+      "debondingDelegations_2": "Delegacije v razvezi ({{count}})",
+      "debondingDelegations_3": "Delegacije v razvezi ({{count}})"
```

Comment:
Was this for testing?

English version: https://github.com/oasisprotocol/oasis-wallet-web/blob/1b059720f94aad0c3bec69599b1d19e03543f3b4/src/locales/en/translation.json#L103-L106

Reply:
No, this is correct. Slovenian has four types of plurals. English only has two.

For this particular case I decided it's better to always show the plural term, so the menu entry remains intact when you switch the wallet/network. The number in parenthesis changes and that's clear enough IMO.

Reply:
Oh, this is weird
```
-1 => activeDelegations_1
 0 => activeDelegations_0
 1 => activeDelegations_1
 2 => activeDelegations_2
 3 => activeDelegations_3
 4 => activeDelegations_3
 5 => activeDelegations_0
 6 => activeDelegations_0
```

Reply:
Is this taken from i18next? It seems correct. Take a female noun, first declension:
```
-1 stopinja
0 stopinj
1 stopinja
2 stopinji
3 stopinje
4 stopinje
5 stopinj
6 stopinj
```